dataverse-webapp/src/main/webapp/resources/css/structure.css

```diff
@@ -537,6 +537,53 @@ table[id$='resultsTable'] td > div > div.file-tags-block {
 /* -------- TABLES -------- */
 div.headerless-table .ui-datatable-tablewrapper table > thead {display:none;}
 
+/* -------- Quill TextEditor -------- */
+.ql-editor p {
+    margin: 0 0 10px;
+    padding: 0;
+}
+
+.ql-snow .ql-editor blockquote {
+    padding: 10px 20px;
+    margin: 0 0 20px;
+    font-size: 17.5px;
+    border-left: 5px solid #eee;
+}
+
+.ql-editor sup {
+    position: relative;
+    font-size: 75%;
+    line-height: 0;
+    vertical-align: baseline;
+}
+
+.ql-editor sub, sup {
+    position: relative;
+    font-size: 75%;
+    line-height: 0;
+    vertical-align: baseline;
+}
+
+.ql-editor sub {
+    bottom: -.25em;
+}
+
+.ql-editor sup {
+    top: -.5em;
+}
+
+.ql-snow .ql-editor code {
+    padding: 2px 4px;
+    font-size: 90%;
+    color: #c7254e;
+    background-color: #f9f2f4;
+    border-radius: 4px;
+}
+
+.ql-snow .ql-editor h1, h2 {
```

Comment:
I don't think that this is what we really want to do. Current rule will apply to 'h2' tag that is not inside editor. We need to repeat the whole selector chain if we want this to apply to `h2` that is inside `ql-editor` and `ql-snow`:
`.ql-snow .ql-editor h1, .ql-snow .ql-editor h2`